frontend/packages/pipelines-plugin/src/components/pipelineruns/status/PipelineRunStatusContent.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { Button, Popover, PopoverPosition } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom-v5-compat';
+import { resourcePathFromModel } from '@console/internal/components/utils';
+import { DASH } from '@console/shared';
+import { PipelineRunModel } from '../../../models';
+import { ComputedStatus, PipelineRunKind } from '../../../types';
+import PipelineResourceStatus from './PipelineResourceStatus';
+import PipelineRunStatusPopoverContent from './PipelineRunStatusPopoverContent';
+import './PipelineRunStatusContent.scss';
+
+type PipelineRunStatusProps = {
+  status: string;
+  pipelineRun: PipelineRunKind;
+  title?: string;
+};
+const PipelineRunStatusContent: React.FC<PipelineRunStatusProps> = ({
+  status,
+  pipelineRun,
+  title,
+}) => {
+  const { t } = useTranslation();
+  const [isPopoverOpen, setIsPopoverOpen] = React.useState(false);
+
+  const PopoverContent = () => {
+    return (
+      <>
+        {isPopoverOpen && (
+          <PipelineRunStatusPopoverContent
+            namespace={pipelineRun.metadata.namespace}
+            link={
+              <Link
+                to={`${resourcePathFromModel(
+                  PipelineRunModel,
+                  pipelineRun.metadata.name,
+                  pipelineRun.metadata.namespace,
+                )}/logs`}
+              >
+                {t('pipelines-plugin~View logs')}
+              </Link>
+            }
```

Comment:
I don't see a reason for the namespace and link prop here since both depends on the pipelineRun that you also pass to the `PipelineRunStatusPopoverContent` component.